api.md

```diff
@@ -1832,29 +1832,75 @@ DROP MATERIALIZED VIEW contagg_view;
 
 ## refresh_continuous_aggregate() :community_function: [](refresh_continuous_aggregate)
 
-Refresh all buckets of a continuous aggregate between two points of
-time. 
+Refresh all buckets of a continuous aggregate in the _refresh window_
+given by `window_start` and `window_end`.
+
+A continuous aggregate materializes aggregates in time buckets (e.g.,
+min, max, average over 10 minutes of data), as determined by the
+continuous aggregate view query's `time_bucket` expression. Therefore,
+when refreshing the continuous aggregate, only buckets that completely
+fit within the refresh window will be refreshed. In other words, it is
+not possible to compute the aggregate over, e.g., half a bucket, and
+any buckets that do no fit within the given refresh window will be
+excluded.
+
+The function expects the window parameter values to have a time type
+that is compatible with the continuous aggregate's time bucket
+expression (e.g., if the time bucket is specified in `TIMESTAMP WITH
+TIME ZONE`, then the start and end time supplied should be a date or
+timestamp type). Note that a continuous aggregate using the `TIMESTAMP
+WITH TIME ZONE` type aligns with the UTC time zone, so any potential
+time zone shifts need to be accounted for when refreshing.
 
-The function expects the parameter values to have the same time type
-as used in the continuous aggregate's time bucket expression (e.g., if
-the time bucket specifies in `timestamptz`, then the start and end time
-supplied should also be `timestamptz`).
 
 #### Required Arguments [](refresh_continuous_aggregate-required-arguments)
 
 |Name|Description|
 |---|---|
 | `continuous_aggregate` | (REGCLASS) The continuous aggregate to refresh. |
-| `window_start` | Start of the window to refresh, has to be before `window_end`. `NULL` is eqivalent to `MIN(timestamp)` of the hypertable. |
-| `window_end` | End of the window to refresh, has to be after `window_start`. `NULL` is eqivalent to `MAX(timestamp)` of the hypertable. |
+| `window_start` | Start of the window to refresh (inclusive), has to be before `window_end`. `NULL` is equivalent to an open-ended start of the window. |
+| `window_end` | End of the window to refresh (exclusive), has to be after `window_start`. `NULL` is equivalent to open-ended end of the window. |
```

Comment:
open-ended start and end of the window is very mathematical/scientific and unlikely be understood by users. What does NULL translate too? Will it be MIN and MAX values of the time type? If so, can it be such specific?

Reply:
Not really limited to science or mathematics, it's in general use for anything that does not have a predetermined limit. See, for example, definition in Cambridge Dictionary (https://dictionary.cambridge.org/dictionary/english/open-ended).

Reply:
I still doubt that many users will understand it.

Reply:
Strictly, saying MAX (exclusive) means the max value is not refreshed. Also, because of bucketing, we might not strictly refresh to the min or max since that might not cover a full bucket. We could still say that we refresh to min or max if that is preferred, but I also fear that this can be misinterpreted as refreshing to the min and max value in the hypertable (which is the reason I changed this formulation).

Reply:
My point is that I don't think it is easy to follow the current formulation.

Reply:
Right now I don't know how to address this since there is only one individual that thinks this is unclear and using min/max as suggested has other drawbacks. No other suggestions have been given either.

Reply:
I guess only me and Mats seen this new formulation. May be @ryanbooz can look to this new version of the formulation.

Reply:
Just based on what you wrote:
```suggestion
| `window_start` | Start of the window to refresh (inclusive), has to be before `window_end`. `NULL` is equivalent to the inclusive start of the first possible bucket for the time dimension. |
| `window_end` | End of the window to refresh (exclusive), has to be after `window_start`. `NULL` is equivalent to the exclusive end of the last possible bucket for the time dimension.. |
```
Will it be correct formulation?